docs/en_US/NNICTLDOC.md

```diff
@@ -567,6 +568,54 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl package show
     ```
 
+<a name="feed"></a>
+![](https://placehold.it/15/1589F0/000000?text=+) `nnictl feed`
+
+* Description
+
+  You can use this command to feed several prior or supplementary trial hyperparameters and results for NNI hyperparameter tuning.
+
+* Usage
+
+  ```bash
+  nnictl feed [OPTIONS]
+  ```
+* Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------|------|
+  |id|  False| |The id of the experiment you want to resume|
+  |--file, -f|  True| |the json file storing your feed data|
+
+* Details
+
+  **When using `nnictl feed`, make sure your data's search space is consistent with the experiment you're going to feed. The json file should be written as a tuple consisting of several trial results. Each results is the dict type including `"parameter"` and `"value"`.**
```

Comment:
The file is in json format, an example is shown below:
  ```json
  [
    {"paramter": {"x": 0.5, "y": 0.9}, "value": 0.03},
    {"paramter": {"x": 0.4, "y": 0.8}, "value": 0.05},
    {"paramter": {"x": 0.3, "y": 0.7}, "value": 0.04}
  ]
  ```
Every element in the top level list is a sample. For our built-in tuners/advisors, each sample should have at least two keys: `parameter` and `value`. The `parameter` must match this experiment's search space, that is, all the keys (or hyperparameters) in `parameter` must match the keys in the search space. Otherwise, tuner/advisor may have unpredictable behavior. `value` should follow the same rule of the input in `nni.report_final_result`, that is, either a number or a dict with a key named `default`. For your customized tuner/advisor, the file could have any json content depending on how you implement the corresponding methods (e.g., `feed_tuning_data`).

Reply:
Done.